.ruff.toml

```
@@ -0,0 +1,37 @@
+target-version = "py310"
+line-length = 110
+exclude = [
+  ".git,",
+  "__pycache__",
+  "build",
+  "ndcube/version.py",
+]
+
+[lint]
+select = ["E", "F", "W", "UP", "PT"]
+extend-ignore = [
```

```suggestion
[lint]
```suggestion
select = [
  "E",
  #"F",
  "W",
  #"UP",
  #"PT"
]
extend-ignore = [
  "E712",
```